Cargo.toml

```diff
@@ -32,9 +32,12 @@ exclude = ["soroban-sdk-macros"]
 [patch.crates-io]
 soroban-sdk = { path = "soroban-sdk" }
 soroban-sdk-macros = { path = "soroban-sdk-macros" }
-soroban-env-guest = { git = "https://github.com/stellar/rs-soroban-env", rev = "21bc98a" }
-soroban-env-host = { git = "https://github.com/stellar/rs-soroban-env", rev = "21bc98a" }
-stellar-xdr = { git = "https://github.com/stellar/rs-stellar-xdr", rev = "28a28dc0" }
+soroban-env-common = { git = "https://github.com/stellar/rs-soroban-env", rev = "556c75f" }
+soroban-env-guest = { git = "https://github.com/stellar/rs-soroban-env", rev = "556c75f" }
+soroban-env-host = { git = "https://github.com/stellar/rs-soroban-env", rev = "556c75f" }
+soroban-env-macros = { git = "https://github.com/stellar/rs-soroban-env", rev = "556c75f" }
+soroban-native-sdk-macros = { git = "https://github.com/stellar/rs-soroban-env", rev = "556c75f" }
```

Comment:
Why is this repo dependent on the `soroban-native-sdk-macros`?

Reply:
It's not, but it's the transitive patch problem. I need to specify the whole set of patches even if they aren't direct dependencies. Because soroban-env-host depends on soroban-native-sdk-macros, this line is required. This annoys me and it would be great if we didn't need to do this.